libcodechecker/analyze/log_parser.py

```diff
@@ -60,9 +60,34 @@ def get_compiler_includes(compiler, extra_opts=None):
                 do_append = True
 
     except OSError as oerr:
-        LOG.error("Cannot find include paths:" + oerr.strerror+"\n")
+        LOG.error("Cannot find include paths: " + oerr.strerror + "\n")
     return include_paths
 
+def get_compiler_target(compiler):
+    """
+    Returns the target triple of the given compiler as a string.
+    """
+    target_label = "Target:"
+    cmd = compiler + " " + "-v"
+    LOG.debug("Retrieving target platform information via '" + cmd + "'")
+    target = ""
+
+    try:
+        proc = subprocess.Popen(shlex.split(cmd),
+                                stdin=subprocess.PIPE,
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE)
+
+        out, err = proc.communicate("")
```

Comment:
Is the output written to `stderr`? If `out` is not used anywhere, don't introduce a variable for it: `_, err = proc.communicate()`.